src/bin/vip-wp.js

```diff
@@ -327,17 +307,19 @@ commandWrapper( {
 			} );
 		} );
 
-		subShellRl.on( 'SIGINT', () => {
-			// Handler here can't be async so all methods need to fire and forget
+		subShellRl.on( 'SIGINT', async () => {
+			//if we have a 2nd SIGINT, exit immediately
```

Comment:
If there are 2 SIGINT events, we'll still be sending it to the API, but just exiting right away in the client side, right? 🤔

Reply:
I've added a return to make sure it doesn't run the rest for whatever reason.  We don't send a second ctrl-C char (but we could). I think on the 2nd try we want to do the bare minimum because of for some reason it errors or skips it the 2nd time, we don't want the user trapped in the command with no way to get out. This is very much an edge case though. It was actually more for my testing purposes when I was reworking some of the sequence and couldn't get out :)

Reply:
Hmmm we definitely don't want to block a user without a way out :). Let's leave it here then.